src/components/layout/MenuBar/MenuBar.module.scss

```diff
@@ -50,3 +53,78 @@
     }
   }
 }
+
+.product_search {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  margin: 20px 0px 5px 0;
+}
+
+.hamburger {
+  position: absolute;
+  top:53%;
+  left: 80%;
+  display:none;
+  margin:0px;
+  padding: 0px
+}
+
+.hamburger_label {
+  display: none;
+  margin:0px;
+  padding: 0px
+}
+
+.icon {
+  width: 35px;
+  height: 35px;
+  cursor: pointer;
+}
+
+  @media (max-width: 520px) {
+  .root {
+    .product_search {
+      flex: 70%;
+    }
+
+    .hamburger_label {
+      display: flex;
+      flex: auto;
+      position: relative;
+      top: 8px;
+    }
+
+    .icon {
+      display: block;
+    }
+
+    .hamburger:checked ~ .menu {
+      visibility: visible;
+      height: 100vh;
```

Comment:
100vh nie jest potrzebne, bo dostajemy pod menu dużą białą przestrzeń. Dobrze jest przetestować jak to wygląda i dobrać odpowiednia wilkość